### PR TITLE
fixes #124 (added meta tags for better SEO)

### LINKED ISSUE #124: The  most common keywords should appear in one or more of the meta-tag-for SEO
## Description
The  most common keywords should appear in one or more of the meta-tags above. The  primary keywords should appear in the meta-tags to help identify the topic of the  webpage to search engines.

systers
projects 
code 
corps 
source 

Fix:
First of all, you must make sure that the page is using the meta-description tags.
Second, you must adjust these tags content in order to include some of the primary keywords displayed above.
## Acceptance Criteria
you must adjust these tags content in order to include some of the primary keywords displayed above

### Update [Required]
- [ ]  index.html
- [ ] code-of-conduct.html
- [ ] contactus.html
- [ ] member-levels.html
- [ ] newcomers.html
- [ ] open-source-workflow.html
- [ ] reporting-guidelines.html
- [ ] gsoc/2009.html
- [ ]  gsoc/2010.html
- [ ]  gsoc/2011.html
- [ ]  gsoc/2012.html
- [ ] gsoc/2014.html
- [ ] gsoc/2015.html
- [ ] gsoc/2016.html
- [ ]  gsoc/2017.html
- [ ] RGSoC/2018.html
- [ ] Outreachy/2017.html

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
0.5 hours

### PR DESCRIPTION
**Pull request by sarnava1**
### Description
This PR solves a long standing issue on meta tags. This PR not only adds meta keywords but also adds all other tags and ogp tags for better SEO.
Fixes #124 

### Type of Change:
**Delete irrelevant options.**

- Code
- Quality Assurance
- Outreach
- Documentation
  

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### Checklist:

**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged 
  

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Comment by sarnava1** [comment]
@bethanyr @ancyp @anumehaagrawal @Rupinderwadali @BaaniLeen  @ExactlyMay @Tharangi please review my PR

File: src/index.html [hunk 1]
```diff
@@ -1,12 +1,29 @@
...
 <html lang="en">
 <head>
-  <meta charset="utf-8">
+  <meta charset="UTF-8" />
   <title>Systers Open Source</title>
   <base href="/">
-
-  <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <meta charset="UTF-8" />
```
Comment (bethanyr) [inline src/index.html]: this line appears to be duplicate of line #4.

File: src/index.html [hunk 2]
```diff
@@ -1,12 +1,29 @@
...
+  <meta content="IE=edge" http-equiv="X-UA-Compatible" />
+  <meta name="keywords" content="slack, slack-ghc, AnitaB.org, systers, code,
```
Comment (bethanyr) [inline src/index.html]: @ExactlyMay - are there any marketing guidelines for the keywords and description for the systers.io website?

File: src/index.html (hunk 1 above)
Comment (sarnava1) [reply src/index.html]: @bethanyr I will remove it.

File: src/index.html (hunk 2 above)
Comment (sarnava1) [reply src/index.html]: Yes if there any such guidelines please do let me know.

**Comment by sarnava1** [comment]
@BaaniLeen @bethanyr I will resolve the conflicts.

**Comment by sarnava1** [comment]
I will do the changes in a new PR and solve the issue.

**Pull request closed without merging**